api/v1/performanceprofile_types.go

```diff
@@ -1,11 +1,29 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package v1
 
 import (
 	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
-
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
```

Comment:
so this comment should probably need to go?

Reply:
sure, autogenerate

Reply:
Autogenerated files are autogenerated once in a while (e.g. bootstrap) or regenerated each time we run some command?
If it is the former case, let's just get rid of the stuff we don't need.

Reply:
done